cornell-dti/flux-functions
# benjamin shen/dining

### Summary 






This pull request is the first step towards implementing menus on Flux.  

- [x] implemented `getAllMenus()` => object containing relevant parts of the Cornell Dining API
- [x] implemented `getEateryMenus(slug)` => array containing all the menus of one eatery
- [x] implemented `getEateryUpcomingMenu(slug,time)` => object with information about one eatery's upcoming menu
- [x] implemented `getEateryDayMenus(slug,time)` => object with information about one eatery's menus throughout the day
- [x] implemented `getAllMealItems(slug,time)` => flattened array containing all the items served in one eatery's upcoming menu
- [x] implemented `getAllItems()` => set containing all the items served on menus this week

### Test Plan 



1. Uncomment `// test();` on line 198
2. Run `node functions/dining` on command line
3. Change slugs and observe console messages

### Notes 



- `dining.js` is currently a standalone file and is not integrated with the rest of Flux or flux-functions

### Breaking Changes
